ui/app/AppLayouts/Chat/views/ChatView.qml

```diff
@@ -177,7 +177,7 @@ StatusAppThreePanelLayout {
         confirmationText: qsTr("Are you sure you want to remove this contact?")
         onConfirmButtonClicked: {
             let pk = chatColumn.contactToRemove
-            if (Utils.getContactDetailsAsJson(pk).isContact) {
+            if (Utils.getContactDetailsAsJson(pk).isAdded) {
```

Comment:
```suggestion
            if (Utils.getUserDetailsAsJson(pk).isAdded) {
```
we should probably create a separate task to fix status-desktup codebase naming in regards to contact vs user